docs/content/preview/yugabyte-platform/troubleshoot/install-upgrade-issues/kubernetes.md

```diff
@@ -0,0 +1,309 @@
+  kubectl describe pods <YBA-POD-NAME> -n <NAMESPACE>
+  ```
+
+- Check the logs for particular container
+
+  ```sh
+  ## yugabytedb anywhere
+  kubectl logs <YBA-POD-NAME> -n <NAMESPACE> -c yugaware
+
+  ## postgres
+  kubectl logs <YBA-POD-NAME> -n <NAMESPACE> -c postgres
+  ```
+
+## CORS error while accessing YBA through LB
+
+You might experience the issue while accessing the YugabyteDB Anywhere through a load balancer. You can define the Cross-Origin Resource Sharing (CORS) domain configuration by setting the [additionAllowedCorsOrigins](https://github.com/yugabyte/charts/blob/master/stable/yugaware/values.yaml#L66) value to the new domain involved. For example, you would add the following to the appropriate Helm command.
+
+  ```properties
+   --set additionAllowedCorsOrigins:'https://mylbdomain'
+  ```
+
+## Load Balancer + EKS
+
+If the load balancer is not working as intended for internet facing, see [EKS + Load Balancer](https://docs.yugabyte.com/stable/yugabyte-platform/install-yugabyte-platform/prepare-environment/kubernetes/).
+
+<!--
+Sometimes the default Amazon Web Services (AWS) load balancer brought up in Amazon Elastic Kubernetes Service (EKS) by the YugabyteDB Anywhere Helm chart is not suitable for your setup, you can use the following settings to customize the [AWS load balancer controller](https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.2/guide/service/annotations/) behavior.
+  - `aws-load-balancer-scheme` can be set to `internal` or `internet-facing` string value.
+  - `aws-load-balancer-backend-protocol` and `aws-load-balancer-healthcheck-protocol` should be set to the `http` string value.
+
+  Consider the following sample configuration:
+
+  ```properties
+  service.beta.kubernetes.io/aws-load-balancer-type: "ip"
+  service.beta.kubernetes.io/aws-load-balancer-scheme: "internet-facing"
+  service.beta.kubernetes.io/aws-load-balancer-backend-protocol: "http"
+  service.beta.kubernetes.io/aws-load-balancer-healthcheck-protocol: "http"
+  ```
+-->
+
+## Is the load balancer in a Pending state?
+
+**Symptom**
+
+You will see the Load Balancer IP assignment in the Pending state. There can be multiple reasons for it.
+
+- Absence of the load balancer controller
+- Exceeds the Public IP quota
+
+**Debug**
+
+- Most cloud providers provide the load balancer controller to serve the load balancer type service. You need to verify whether the cluster has the load balancer controller.
+
+**Resolution**
+
+- In case of the absence of a load balancer controller in GKE, follow the [GKE Ingress Guide](https://cloud.google.com/kubernetes-engine/docs/concepts/ingress#overview).
+
+- You might experience Public IP not associated with the load balancer if you exceed the Public IP quota limit.
+
+- In case of Minikube, make sure to run the [minikube tunnel](https://minikube.sigs.k8s.io/docs/commands/tunnel/).
+
+
+## Unable to expand PVC
+
+**Symptom**
+
+You will see the error related to unable to expand the PVC.
```

Comment:
I think it would be better to show the exact error, where to see the detailed error in events, logs, what command to run, etc. Similar to other sections we have.